user_docs/en/userGuide.t2t

```diff
@@ -65,46 +72,223 @@ This applies to both original and modified copies of this software, plus any der
+"Basic Training for NVDA" is available from the [NV Access Shop http://www.nvaccess.org/shop]
 
-++ Installing NVDA ++[InstallingNVDA]
-If installing NVDA directly from the NVDA download package, press the Install NVDA button.
-If you have already closed this dialog or are wanting to install from a portable copy, please choose the Install NVDA menu item found under Tools in the NVDA menu.
+Here are some basic commands which are used frequently.
+All commands are configurable, so these are the default keystrokes for these functions.
+
++++ The NVDA Modifier Key
+The default NVDA modifier key is either the `numpad zero`, (with `numlock` off), or the `insert` key, near the `delete`, `home` and `end` keys.
+The NVDA modifier key can also be set to the `caps lock` key.
 
-The installation dialog that appears will confirm whether you wish to install NVDA and will also tell you whether  this installation will be updating a previous install.
++++ Input Help
+To learn and practice the location of keys, press `NVDA+1` to turn Input help on.
+While in input help mode, performing any input gesture (such as pressing a key or performing a touch gesture) will report the action and describe what it does (if anything).
+The actual commands will not execute while in input help mode. 
+
++++ Starting and stopping NVDA:
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Start NVDA | control+alt+n | control+alt+n | | Starts or restarts NVDA |
+| Exit NVDA | NVDA+q, then enter | NVDA+q, then enter | | Exits NVDA |
+| Pause or restart speech | shift | shift | | Instantly pauses speech. Pressing it again will continue speaking where it left off |
+| Stop speech | control | control | | Instantly stops speaking |
+
++++ Reading text:
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Say all | NVDA+down arrow | NVDA+a | | Starts reading from the current position, moving it along as it goes |
+| Read current line | NVDA+up arrow | NVDA+l | | Reads the line. Pressing twice spells the line. Pressing three times spells the line using character descriptions (Alpha, Bravo, Charlie, etc) |
+| Read selection | NVDA+shift+up arrow | NVDA+shift+s | | Reads any selected text |
+| Read clipboard text | NVDA+c | NVDA+c | | Reads any text on the clipboard |
+
++++ Reporting location and other information:
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Window title | NVDA+t | NVDA+t | | Reports the title of the currently active window. Pressing twice will spell the information. Pressing three times will copy it to the clipboard |
+| Report focus | NVDA+tab | NVDA+tab | | Reports the title of the currently active window. Pressing twice will spell the information. Pressing three times will copy it to the clipboard |
+| Read window | NVDA+b | NVDA+b | | Reports the title of the currently active window. Pressing twice will spell the information. Pressing three times will copy it to the clipboard |

```

Comment:
The description here is incorrect.

Reply:
So it was, for some reason it didn't copy the text when I selected the other descriptions so pasted the same thing again.  I've updated it now.